apps/files/src/mixins.js

```diff
@@ -346,7 +346,7 @@ export default {
           let p
 
           if (this.publicPage()) {
-            p = this.queue.add(() => this.$client.publicFiles.createFolder(this.rootPath + directory))
+            p = this.queue.add(() => this.$client.publicFiles.createFolder(this.rootPath, directory, this.publicLinkPassword))
```

Comment:
Keep the + between rootPath and directory pls

Reply:
Also, this function is expecting also token in case you need to specify password - https://github.com/owncloud/owncloud-sdk/blob/f232b49e6a6ca6ef0a20ce0fb256948d1cfed38b/src/publicLinkAccess.js#L112

Reply:
Can you please explain the reason?
Doesn't the following already add that?

https://github.com/owncloud/owncloud-sdk/blob/f232b49e6a6ca6ef0a20ce0fb256948d1cfed38b/src/publicLinkAccess.js#L118

Reply:
Or does the rootPath actually contains the token here instead of the folder?

Reply:
rootPath is the token.

Reply:
Ah, okay then. I thought it's the same as in non-public link view. Then it's fine :+1:

Reply:
`rootPath` is the token. It is stored similar to "d325vIQcteMyyW7/". What should I do?